src/external/lua/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.6)
```

Comment:
Ok, Solaris I understand, but an Ubuntu LTS is no reason for me in this case as we will release way after that LTS got unsupported.